sddir.py

```diff
@@ -28,8 +29,16 @@
     "img.huffingtonpost.com",
     "disclose.ngo",
     "techcrunch.com",
-    "tv2.dk",
     "webapps.aljazeera.net",
+    "www.apache.be",
```

Comment:
These two new exceptions are the only ones prefixed with www - is that intentional?

Reply:
Just a coincidence, these entries have to match how they're listed in onboarded.txt: https://github.com/freedomofpress/securedrop-https-everywhere-ruleset/blob/470ac857ae1c320794f46e3be0bede206e92e10e/onboarded.txt

Reply:
Ah, makes sense - thanks for confirming!